core/tests/protractor_utils/workflow.js

```diff
@@ -57,14 +57,22 @@ var checkForAddTitleWarning = async function() {
 
 // Trigger onblur event for title.
 var triggerTitleOnBlurEvent = async function() {
-  await element(by.css('.protractor-test-exploration-title-input')).click();
-  await element(by.css('.protractor-test-exploration-objective-input')).click();
+  var testExplorationTitleInput = element(
+    by.css('.protractor-test-exploration-title-input'));
+  await action.click('Test Exploration Title Input', testExplorationTitleInput);
+  var testExplorationObjectiveInput = element(
+    by.css('.protractor-test-exploration-objective-input'));
+  await action.click(
+    'Test Exploration Objective Input', testExplorationObjectiveInput);
 };
 
 // Open edit roles.
 var openEditRolesForm = async function() {
-  await element(by.css('.protractor-test-edit-roles')).click();
-  await element(by.css('.protractor-test-role-username')).sendKeys(
+  var testEditRoles = element(by.css('.protractor-test-edit-roles'));
+  await action.click('Test Edit Roles', testEditRoles);
+  await action.sendKeys(
+    'Username input',
+    element(by.css('.protractor-test-role-username')),
     'Chuck Norris');
```

Comment:
When breaking the contents of parentheses across lines, the open parenthesis should end its line

```suggestion
  await action.sendKeys(
    'Username input',
    element(
      by.css('.protractor-test-role-username')),
      'Chuck Norris');
```